configure.ac

```diff
@@ -720,6 +762,29 @@ AC_LINK_IFELSE([AC_LANG_SOURCE([
 )
 LDFLAGS="$TEMP_LDFLAGS"
 
+TEMP_LDFLAGS="$LDFLAGS"
+LDFLAGS="$TEMP_LDFLAGS -mpower8-vector"
+AC_MSG_CHECKING([for POWER8 compiler support])
+AC_LINK_IFELSE([AC_LANG_SOURCE([[
+  #include <altivec.h>
+  #include <stdint.h>
+  int main(){
+  unsigned char src[16];
+  __builtin_crypto_vshasigmaw((__vector uint32_t)vec_vsx_ld(0, src), 1, 0xf);
+  return 0;
+  }
+  ]])],
+  [
+    AC_DEFINE(HAVE_POWER8,1,[Define if compiler supports POWER8 instructions.])
```

Comment:
Please move the defines/conditionals down with the other asm flags for consistency.